dist/angular-auth.min.js

```diff
@@ -1,5 +1,5 @@
 angular.module("PlAuth",["restmod"]);
-!function(){"use strict";function t(){function t(){this.getLocalStoragePrefix=function(){return u},this.getUidHeaderName=function(){return i},this.getTokenHeaderName=function(){return r},this.getUidKey=function(){return e},this.getTokenKey=function(){return n}}var e="uid",n="token",i="X-User-Email",r="X-User-Token",u="angularAuth";this.setLocalStoragePrefix=function(t){u=t},this.setTokenHeaderName=function(t){r=t},this.setUidHeaderName=function(t){i=t},this.setUidKey=function(t){e=t},this.setTokenKey=function(t){n=t},this.$get=function(){return new t}}angular.module("PlAuth").provider("AuthConfig",t)}();
-!function(){"use strict";function e(e,t){function r(){return!!e.getKey("uid")&&!!e.getKey("token")}function n(){return!!e.getKey("uid")}function i(){var t={};return t[g]=e.getKey("uid"),t[d]=e.getKey("token"),t}function o(t){if(!angular.isObject(t))throw new Error("an object was expected to store");if(t[s]){if(!t[c])throw new Error("A uid is need");e.setKey("token",t[s])}t[c]&&e.setKey("uid",t[c])}function u(){e.eraseKey("uid"),e.eraseKey("token")}var a={store:o,clear:u,isLogged:r,isRegistered:n,getAuthorizationHeaders:i},d=t.getTokenHeaderName(),g=t.getUidHeaderName(),s=t.getTokenKey(),c=t.getUidKey();return a}angular.module("PlAuth").service("AuthSrv",e),e.$inject=["LocalDataSrv","AuthConfig"]}();
+!function(){"use strict";function t(){function t(){this.getLocalStoragePrefix=function(){return r},this.getUidHeaderName=function(){return i},this.getTokenHeaderName=function(){return u},this.getUidKey=function(){return e},this.getTokenKey=function(){return n},this.getUidRequired=function(){return o}}var e="uid",n="token",i="X-User-Email",u="X-User-Token",r="angularAuth",o=!0;this.setLocalStoragePrefix=function(t){r=t},this.setTokenHeaderName=function(t){u=t},this.setUidHeaderName=function(t){i=t},this.setUidKey=function(t){e=t},this.setTokenKey=function(t){n=t},this.setUidRequired=function(t){o=t},this.$get=function(){return new t}}angular.module("PlAuth").provider("AuthConfig",t)}();
```

Comment:
Line is too long.<br>Possible strict violation.<br>Missing semicolon.<br>Expected an assignment or function call and instead saw an expression.<br>'e' was used before it was defined.<br>'n' was used before it was defined.<br>'i' was used before it was defined.<br>'u' was used before it was defined.<br>'r' was used before it was defined.<br>'o' was used before it was defined.<br>A constructor name should start with an uppercase letter.<br>Missing '()' invoking a constructor.<br>Wrap an immediate function invocation in parens to assist the reader in understanding that the expression is the result of a function, and not the function itself.